sdk/ai/ai-projects/src/customization/convertParametersToWire.ts

```diff
@@ -77,3 +88,12 @@ export function convertListFilesQueryParamProperties(
     ...(options.purpose && { purpose: options.purpose }),
   };
 }
+
+export function convertPollingOptions<T extends PollingOptionsParams>(
+  options: T,
+): Record<string, unknown> {
+  return {
+    ...(options.pollingOptions?.sleepIntervalInMs && { polling_interval: options.pollingOptions?.sleepIntervalInMs }),
+    ...(options.pollingOptions?.abortSignal && { poll_until: options.pollingOptions?.abortSignal }),
+  };
+}
```

Comment:
this is not needed as its not being passed over the wire its need for the client